baseplate/server/healthcheck.py

```diff
@@ -0,0 +1,74 @@
+"""Check health of a baseplate service on localhost."""
```

Comment:
I know you're just moving this script, so maybe it's out of scope for this PR, but it'd be nice to have an explanation of what the health checks do, how they should be run, what the output is when they fail, and how they are intended to be used.